features/steps/space_steps.js

```diff
@@ -3,17 +3,62 @@ const { RoomPage, SpacePage, SpaceEditPage } = require("../harness/Pages");
 const { linkParameters } = require("../lib");
 const appUrl = require("../lib/appUrl");
 const { Api } = require("../lib/Api");
+const AuthenticationMethod = require("../lib/AuthenticationMethod");
+const SpaceMembership = require("../lib/SpaceMembership");
+const crypto = require("crypto");
 
 Given("{a} {space}", function (_, space) {
   const api = new Api(appUrl(), process.env.OPERATOR_API_KEY);
 
-  return api.spaces().create(space);
+  this.spaces = this.spaces || {};
+
+  return api
+    .spaces()
+    .create(space)
+    .then((space) => (this.spaces[space.name] = space));
 });
 
-Given("{a} {space} has {a} {actor}", function (_, space, _, actor) {
-  return true;
+Given("a Space with a Room", function () {
+  // This space intentionally left blank... For now...
+  // TODO: Create a Space for each test instead of re-using the
+  //       System Test Space
 });
 
+Given(
+  "{a} {space} has {a} {actor}",
+  /**
+   *
+   * @param {*} _
+   * @param {Space} space
+   * @param {*} _
+   * @param {Actor} actor
+   * @returns
+   */
+  function (_, space, _, actor) {
+    this.actors = this.actors || {};
+    this.actors[actor.email] = actor;
+
+    space = this.spaces[space.name] || space;
+    actor.email = `${this.testId()}+${actor.email}`;
+    const api = new Api(appUrl(), process.env.OPERATOR_API_KEY);
+    const toCreate = new AuthenticationMethod({
+      contactMethod: "email",
+      contactLocation: actor.email,
+    });
+
+    return api
+      .authenticationMethods()
```

Comment:
Let's swap this to `api.people().create(...)` and create the Person, rather than an `AuthenticationMethod`